### PR TITLE
Add site-wide banner: NG vers is v5

### PR DESCRIPTION
**Pull request by chalin**
Staged at https://angulardart-org-dev3.firebaseapp.com

### Screenshots

> image

> image

### CODE REVIEW

File: src/_includes/banner.html [hunk 1]
```diff
@@ -31,7 +31,8 @@
   {%- else -%}
     <p class="banner__text">
       <i class="material-icons"> new_releases </i>
-      This site covers <b>web development</b> with <b>Dart 2</b>.
+      This site covers <b>web development</b> with <b>Dart 2</b>
```
Comment (kwalrath) [inline src/_includes/banner.html]: Ha, I just realized the **Dart 2** is in there. I think we can drop that. :)

...with Angular 5

Comment (chalin) [reply src/_includes/banner.html]: Done

**Comment by kwalrath** [comment]
We should simplify the versions page. No need to list previous and future versions, I think.

**Comment by chalin** [comment]
Versions page updates would be covered by #105.

**Pull request merged**